### PR TITLE
More descriptive banners for experiments, and allow adjusting FF distribution

### PR DESCRIPTION
**Pull request by neilkakkar**
## Changes

Updated based on @clarkus designs:

Completed Experiment with Significant results:

image

Completed Experiment with non-significant results:

image

---

Running experiment, not significant:

image

image

## How did you test this code?

Manual QA, see things look good

### CODE REVIEW

**Review by clarkus — CHANGES_REQUESTED** [review]
Looks good to me. Can we remove the line break between the opening line and the reason line for some banners? It just makes it kind of hard to read when there's an unexpected newline. Feels like it should just be all one non-breaking paragraph. Thoughts?

**Comment by neilkakkar** [comment]
Can go back to that. I found it harder to parse / too-much-in-a-paragraph-to-take-in which is why I broke it down, but no strong feelings here, happy to change

File: frontend/src/scenes/experiments/Experiment.scss [hunk 1]
```diff
@@ -186,20 +186,17 @@
...
         border-radius: 4px;
-        text-align: center;
         margin-bottom: $default_spacing;
     }
 
     .feature-flag-mods {
         padding: $default_spacing / 2;
         width: 100%;
-        background: rgba($brand_yellow, 0.1);
+        background: #f6ebf8;
```
Comment (paolodamico) [inline frontend/src/scenes/experiments/Experiment.scss:197]: Seems pretty similar to `$purple_300`, should we use that for consistency @clarkus ?

File: frontend/src/scenes/experiments/Experiment.tsx [hunk 2]
```diff
@@ -533,6 +541,18 @@ export function Experiment_(): JSX.Element {
...
                                     </Tag>
+                                    {experimentResults && experimentData.end_date && (
+                                        <Tag
+                                            style={{ alignSelf: 'center' }}
+                                            color={areResultsSignificant ? 'green' : 'geekblue'}
+                                        >
+                                            <b className="uppercase">
+                                                {areResultsSignificant
+                                                    ? 'Significant Results'
+                                                    : 'Results not significant'}
```
Comment (paolodamico) [inline frontend/src/scenes/experiments/Experiment.tsx:552]:
```suggestion
                                                    : 'Results not yet significant'}
```

File: frontend/src/scenes/experiments/Experiment.tsx [hunk 3]
```diff
@@ -594,21 +614,40 @@ export function Experiment_(): JSX.Element {
...
-                                    {significanceTooltip && (
-                                        <Tooltip placement="right" title={significanceTooltip}>
-                                            <InfoCircleOutlined style={{ color: '#f96132', padding: '4px 2px' }} />
-                                        </Tooltip>
-                                    )}
+                                <Col span={23} style={{ color: '#2D2D2D' }}>
+                                    <b>Your results are not statistically significant</b>. {significanceBannerDetails}{' '}
+                                    {experimentData?.end_date ? '' : "We don't recommend ending this experiment yet."}{' '}
+                                    See our{' '}
+                                    <a href="https://posthog.com/docs/user-guides/experimentation#funnel-experiment-calculations">
```
Comment (paolodamico) [inline frontend/src/scenes/experiments/Experiment.tsx:623]: Is "#funnel-experiment-calculations" relevant for trends too?

**Review by paolodamico — APPROVED** [review]
lgtm

File: frontend/src/scenes/experiments/Experiment.tsx (hunk 2 above)
Comment (neilkakkar) [reply frontend/src/scenes/experiments/Experiment.tsx:552]: This is shown on completion only, so shouldn't have "yet" in it.

File: frontend/src/scenes/experiments/Experiment.tsx (hunk 3 above)
Comment (neilkakkar) [reply frontend/src/scenes/experiments/Experiment.tsx:623]: Just the heading below, visible on the same page, so should be ok

**Pull request merged**